include/roboteam_ai/treeinterp/BTFactory.h

```diff
@@ -52,8 +54,7 @@ class BTFactory {
     static std::string currentTree;
     static std::string keeperTree;
     static bool weMadeTrees;
-
-    static std::shared_ptr<bt::BehaviorTree> testing_tree;
+    static std::shared_ptr<rtt::ai::analysis::Play> play;
```

Comment:
What I don't like is that we essentially have a set of x functions that's just using a class as a namespace, turn it into a singleton if you want it as a global, or change it to loose functions

Reply:
Boi ill sacrifice myself and refactor BT "Factory" (this ain't java)

Reply:
Hahaha pretty soon we won't need it anymore. We actually don't need it anymore at all, but I didn't wanna get rid of it completely since it's still used by the interface etc

Reply:
If you want to refactor it then feel welcome, but we should discuss it first as about 70% of the code is not useful anymore in a week or 2